ranger/data/scope.sh

```diff
@@ -293,20 +294,20 @@ handle_mime() {
         ## Text
         text/* | */xml)
             ## Syntax highlight
-            if [[ "$( stat --printf='%s' -- "${FILE_PATH}" )" -gt "${HIGHLIGHT_SIZE_MAX}" ]]; then
+            if [[ "$(stat --printf='%s' -- "${FILE_PATH}")" -gt "${HIGHLIGHT_SIZE_MAX}" ]]; then
                 exit 2
             fi
-            if [[ "$( tput colors )" -ge 256 ]]; then
+            if [[ "$(tput colors)" -ge 256 ]]; then
                 local pygmentize_format='terminal256'
                 local highlight_format='xterm256'
             else
                 local pygmentize_format='terminal'
                 local highlight_format='ansi'
             fi
-            env HIGHLIGHT_OPTIONS="${HIGHLIGHT_OPTIONS}" highlight \
```

Comment:
Calling env is more portable so I see no reason not to.

Reply:
Well, it's one more external process. Is there anything that doesn't support the non-`env` variant? I tested on `sh`, `bash`, and `zsh`.

Reply:
Fish and csh : )
If you want a fast `scope.sh` you could write one in C or Go or any language of your choosing.